packages/regl-worldview/src/index.js

```diff
@@ -16,6 +16,7 @@ export { default as fromGeometry } from "./utils/fromGeometry";
 export * from "./utils/Raycast";
 export * from "./commands/index";
 export * from "./types/index";
+export { default as WorldviewReactContext } from "./WorldviewReactContext";
```

Comment:
Cool, I was just thinking this might be handy in a command I'm working on (specifically the ability to call `onDirty()`)